project/src/main/java/com/google/sps/Servlets/UserServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+import com.google.gson.*;
+
+@WebServlet("/User")
+public class UserServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      HttpSession session = request.getSession(true);
+
+      String userEmail = userService.getCurrentUser().getEmail();
```

Comment:
Is it possible to also get the user's name from the Google auth API

Reply:
I'll look into it, right now I just wanted to get the basic functionality down. If it's not we can always just grab the part of the email before the '@' and use that or prompt the user to pick one.